python/letsql/backends/let/hotfix.py

```diff
@@ -40,7 +43,20 @@ def op(self):
 
     @property
     def cached_nodes(self):
-        return self.op.find(CachedNode)
+        def _find(node):
+            cached = node.find((CachedNode, RemoteTable))
+            if not cached:
+                yield None
+            else:
+                for no in cached:
```

Comment:
this function can start from `for no in node.find((CachedNode, RemoteTable))`, removing what is above, and then you can remove the `filter(None, ...`